plugin.js

```diff
@@ -2,13 +2,20 @@
 
 const fp = require('fastify-plugin')
 const cookie = require('cookie')
+const cookieSignature = require('cookie-signature')
+const cookieSignatureSecret = Symbol('fastify-cookie.secret')
 
 function fastifyCookieSetCookie (name, value, options) {
   const opts = Object.assign({}, options || {})
   if (opts.expires && Number.isInteger(opts.expires)) {
     opts.expires = new Date(opts.expires)
   }
 
+  if (opts.signed) {
+    value = cookieSignature.sign(value, this[cookieSignatureSecret])
+    delete opts.signed
```

Comment:
Can you avoid deleting this property?

Reply:
i think no, this is to avoid pass it to cookie parser.
it's possible add another parameter to the function, maybe?

Reply:
Adding a third parameter?

Reply:
no, just remove the delete.